ui/src/layout/Findings/Vulnerabilities/VulnerabilityDetails.js

```diff
@@ -31,12 +30,6 @@ const DetailsContent = ({data}) => {
                     title: "Asset details",
                     path: VULNERABILITY_DETAILS_PATHS.ASSET_DETAILS,
                     component: () => <AssetDetails assetData={asset} withAssetLink />
-                },
-                {
```

Comment:
(General comment) I see that we now detach the finding from the scan. Don't know if worth removing it when it is associated with a scan. Or even better, should we associate it with the asset scan?